var/spack/repos/builtin/packages/sph2pipe/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Sph2pipe(Package):
+    """Sph2pipe is a portable tool for 
+    converting SPHERE files to other formats."""
+
+    homepage = "https://www.ldc.upenn.edu/language-resources/tools/sphere-conversion-tools"
+    url      = "https://www.ldc.upenn.edu/sites/www.ldc.upenn.edu/files/ctools/sph2pipe_v2.5.tar.gz"
+
+    version('2.5', '771d9143e9aec0a22c6a14e138974be2')
+
+    def install(self, spec, prefix):
+        bash = which('bash')
+        bash('-c', env['CC'] + ' -O2 -o sph2pipe *.c -lm')
```

Comment:
Can we run the C compiler directly instead of through bash?  Alternately, can we just write a CMake file for this package and include it as a patch?